src/analysis/parameter.rs

```diff
@@ -67,5 +69,16 @@ pub fn analyze(env: &Env, par: &library::Parameter, configured_functions: &[&Fun
         ref_mode: ref_mode,
         is_error: par.is_error,
         to_glib_extra: String::new(),
+        is_into: *nullable && !par.instance_parameter && match *env.library.type_(par.typ) {
```

Comment:
Filename is already generated with `Into`, so trying to put it again is a bad idea (the generated code doesn't compile actually).